docs/specs/clients/notify/client-sdk-api.md

```diff
@@ -83,7 +83,9 @@ abstract class Client {
   
   // Returns true if account has a valid and up to date registration, false otherwise
   public abstract isRegistered(params: {
-    account: string
+    account: string,
+    allApps: boolean,
```

Comment:
Should this have the same default as above?

```suggestion
    // Defaulted to false if not passed
    allApps?: boolean,
```

Reply:
I don't think it should be optional personally - you're checking for permissions essentially. 

Is `account` registered under `domain` to access a single app or `allApps`

Reply:
I don't see your argument, I don't see anything wrong with having a default in what you're checking for. I don't want 90% of dapp developers to have to set this to `false` when they don't for the other one.

Reply:
I think that Kotlin and Swift clients it's better to set this to true instead of false

Reply:
Default true on Swift and Kotlin makes sense to me since those are pretty much exclusive to wallets, but dapps are still possible. But for JS it's more tricky because there are React Native wallets.

I would almost rather not have a default, than to default to true.

Also once we pick a default, we cannot change it even through a breaking change because you risk people not reading the migration doc and just upgrading major versions.